src/components/elements/table/hooks/useTableSelection.tsx

```diff
@@ -0,0 +1,72 @@
+import { without } from 'lodash-es';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+
+export function useTableSelection<T extends { id: string }>({
```

Comment:
It looks like something here broke the table selection functionality in the household submit summary page.
I'm no longer able to click the checkboxes to select the row:

<img width="876" alt="Screenshot 2025-02-06 at 12 07 14 PM" src="https://github.com/user-attachments/assets/3ac95a0f-d261-4493-9ada-bac03393ce0b" />

Reply:
mmm nevermind, this issue is on QA too. Row 543 of GenericTable should change `selectable === 'checkbox' && isSelectable ? ` to just `isSelectable ?`, I believe.. would you mind dropping in that change? thank you!

Reply:
ok sorry amending again: it looks like this is a bug newly introduce on release-151, and not present on release-150. So it should be opened as its own fix against 151!

Reply:
https://github.com/greenriver/hmis-frontend/pull/1047

Reply:
thanks!